build/WFLY-16863_Std_configs_galleon_layers.adoc

```diff
@@ -0,0 +1,326 @@
+= WFLY-16863, express standalone default configurations with Galleon Layers, phase 1
+:author:           Jean-Francois Denise
+:email:             jdenise@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Standalone default configurations are wildly used. These configurations allow to start using WildFly without having to customize the server.
+
+Today standalone configurations, due to the Galleon internal structure they are based on, require that a complete installation of the server be installed
+to be usable. They are not benefiting from Galleon trimming capabilities that tooling based on Galleon offers.
+
+Tuning a default configuration to add/remove subsystems is a complex task (mainly due to capability dependencies checks). 
+
+So today, standalone configurations are hard to customize and require a full server to be usable. Although being generated using Galleon they are
+not benefiting from Galleon full capabilities, such as:
+
+* Ability to reduce the server installation size and number of binary artifacts. 
+* Ability to easily add/remove subsystems (and dependencies).
+
+We are here proposing to partially solve these issues by changing the internal Galleon structure they depend on.
+
+Galleon Layers feature has been introduced in Galleon in order to solve this kind of issues. 
+But until now, the standalone default configurations have not been benefiting from it.
+
+We plan to handle this RFE in 2 iterations. 
+
+* A first iteration (this WFLY-16863) that targets WF28 that would re-implement default standalone configuration using the set of existing Galleon layers.
```

Comment:
@jfdenise Can you please add a security considerations section to this proposal? Since this will include re-implementing of default standalone configuration by using existing Galleon layers, even if there is no change in the content of the layes, we need to make sure that the new default behaviour will not accidentally leave an unsecure server

Reply:
@Skyllarr , good suggestion. I added it.